test/hist.jl

```diff
@@ -26,7 +27,6 @@ end
 
 
 @testset "Histogram append" begin
-    # FIXME: closed (all lines in this block):
```

Comment:
Do you have an idea why this block is marked, while the `closed` argument must have been present in the first place?

Reply:
I probably put it in by mistake, back then. I had lots of these `FIXME: closed` comments in the test file already, but then I restructured the test file into testsets when I added normalization. I guess that's how it snuck into that block, even though here `:left` can't be left out (to test with different values for `isdensity`).